src/clj/clojure/core.clj

```diff
@@ -6684,15 +6684,15 @@
   [x] (instance? java.util.UUID x))
 
 (defn reduce
-  "f should be a function of 2 arguments. If val is not supplied,
```

Comment:
Apologies for the hard wrap diff gore. I changed the first sentence from:

> f should be a function of 2 arguments.

to:

> f should be a function of 2 arguments: 1st, the accumulator, and 2nd the current element.